src/components/vue-form-builder.vue

```diff
@@ -67,6 +67,11 @@
 
         <hr class="w-100">
       </b-input-group>
+
+        <b-card-body class="d-flex justify-content-center align-items-center drag-placeholder text-center position-absolute" v-if="!containsFormElements">
```

Comment:
Nitpick: This is a `b-card-body` component but it's not inside of a card. I removed the `card-body` class and it still looks the same, so I think it's better to use a regular div here.

Also, this block is indented one more than the blocks around it–it should in on the same line.